src/lib.rs

```diff
@@ -72,6 +72,9 @@ pub mod from_py;
 pub mod to_py;
 pub mod wrappers;
 
+#[cfg(feature = "f64")]
```

Comment:
Did you consider this being a different feature flag?

Reply:
I used the f64 feature as usescase of geopandas use the f64 coordinates anyways. The second thought was that this implementation would not cause much compilation overhead anyways, so I omitted creating an  own feature for it.